src/app/services/student.service.ts

```diff
@@ -25,4 +25,8 @@ export class StudentService {
   public update(student: Student): Observable<Object> {
     return this.http.put(`${this.apiURL}/${student._id}`, student);
   }
+
+  public delete(_id): Observable<Object> {
```

Comment:
you can use `id`, not needed to name parameters like it is on model or on a table in BE, you can name it gives a meaningful name `sudentId`, `id`, `data` etc.